crypto/sha/asm/keccak1600-ppc64.pl

```diff
@@ -409,10 +409,10 @@
 	.long	0
 .size	dword_le_load,.-dword_le_load
 
-.globl	SHA3_absorb
-.type	SHA3_absorb,\@function
+.globl	SHA3_absorb_vsx
+.type	SHA3_absorb_vsx,\@function
```

Comment:
??? So that **both** assembly modules export SHA3_absorb_vsx... Either way, it makes lesser sense to deploy vector keccak1600p8-ppc.pl, because benefits are not that clear. It's only nominally faster than scalar module on POWER8 but significantly slower on POWER9. Because POWER9 is better equipped to handle non-vector code. On related note, there is version optimized for little-endian, as well as 32-bit version. [And since MIPS was mentioned, there is even MIPS module...]

Reply:
Ok, I will disable it instead

Reply:
I have disabled it in the NetBSD build, but I am not familiar with the autoconfiguration mechanism to disable it in the standard build.